src/glossary.md

```diff
@@ -0,0 +1,39 @@
+# Glossary
+
+## Toronto Community Network
+
+The name of the Toronto-based community mesh network initiative started by Toronto Mesh with its collaborators in 2020.
+
+This book documents the current resources of this initiative. For more information, visit [the project page](https://github.com/tomeshnet/toronto-community-network/).
+
+## Toronto Mesh
+
+A community group that helps communities create better networks with open source and peer-to-peer technologies that promote digital literacy and privacy.
+
+The group that started the Toronto Community Network initiative and its steward for the foreseeable future. For more information, visit [tomesh.net](https://tomesh.net).
+
+## Point-to-point (PTP)
+
+A **PTP** link is a connection that only involves two antennas. These kinds of antennas are more powerful and can reach longer distances, but only one other antenna can connect to them. They're usually implemented using highly directional (i.e. high gain) antennas and used to connect two supernodes.
```

Comment:
This is assuming wireless.  PTP also refers to entities likes Tunnels and vpns
Example: tun0: flags=4305<UP,**_POINTOPOINT_**,RUNNING,NOARP,MULTICAST>  mtu 1500

Also PTP does not define the power of the antenna.  The current antennas we have can be both PTP and PTMP.  The only differnce between the two is the speed as PTP can do 80Mhz while PTMP can only do 40.

The power of the antennas is defined by the geometry of the beam not  PTP vs  PTMP

IE: the LAP 120 with a 120 degree beam can act as PTP mode or PTMP mode, so can the LiteBeam whith only a 24 degree beam.

Reply:
```suggestion
A **PTP** (Point To Point) link is a connection limited to only two devices.  This could be a virtual connection like a VPN or Tunnel, or a physical connection like two routers connected via a fibre optic link, or two antennas configured for P2P.  Although most antennas are capable of doing both PTP and PTMP,  antennas used for PTP usually are directional (ie high gain) that allows them to reach longer distances with less noise.